metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-mongodb",
-  "version": "4.1.2-rc0",
+  "version": "4.2.1-rc0",
```

Comment:
please keep it at 4.2.0. After this PR is merged, someone with write permissions to the repository runs the `release` rake task. That checks if the version isn't a release candidate (no -rc). Only then the release is made and afterwards the version is bumped to the next patch level and rc0 is appended. The goal of this is to prevent accidental releases.